brainscore/benchmarks/__init__.py

```diff
@@ -73,11 +73,15 @@ def __init__(self):
         self['dicarlo.Majaj2015.V4-rdm'] = LazyLoad(DicarloMajaj2015V4RDM)
         self['dicarlo.Majaj2015.IT-rdm'] = LazyLoad(DicarloMajaj2015ITRDM)
         from .freemanziemba2013 import MovshonFreemanZiemba2013V1PLS, MovshonFreemanZiemba2013V2PLS, \
-            MovshonFreemanZiemba2013V1RDM, MovshonFreemanZiemba2013V2RDM
+            MovshonFreemanZiemba2013V1RDM, MovshonFreemanZiemba2013V2RDM, MovshonFreemanZiemba2013V1SINGLE, \
+            MovshonFreemanZiemba2013V1PLSPrivate, MovshonFreemanZiemba2013V1PLSPublic
         self['movshon.FreemanZiemba2013.V1-pls'] = LazyLoad(MovshonFreemanZiemba2013V1PLS)
         self['movshon.FreemanZiemba2013.V2-pls'] = LazyLoad(MovshonFreemanZiemba2013V2PLS)
         self['movshon.FreemanZiemba2013.V1-rdm'] = LazyLoad(MovshonFreemanZiemba2013V1RDM)
         self['movshon.FreemanZiemba2013.V2-rdm'] = LazyLoad(MovshonFreemanZiemba2013V2RDM)
+        self['movshon.FreemanZiemba2013.V1-single'] = LazyLoad(MovshonFreemanZiemba2013V1SINGLE)
+        self['movshon.FreemanZiemba2013.private.V1-pls'] = LazyLoad(MovshonFreemanZiemba2013V1PLSPrivate)
+        self['movshon.FreemanZiemba2013.public.V1-pls'] = LazyLoad(MovshonFreemanZiemba2013V1PLSPublic)
```

Comment:
delete, public benchmarks go to https://github.com/brain-score/brain-score/blob/master/brainscore/public_benchmarks.py